code/src/components/product-card/ProductCard.tsx

```diff
@@ -4,45 +4,45 @@ import AppBox from "@/components/app-box/AppBox";
 import AppButton from "@/components/app-button/AppButton";
 import AppLink from "@/components/app-link/AppLink";
 import AppTypography from "@/components/app-typography/AppTypography";
+import { ProductCardProps } from "@/components/product-card/ProductCard.types";
 
-import { Product } from "@/types/product.types";
 import formatPrice from "@/utils/format-price/formatPrice";
 
 import "@/components/product-card/ProductCard.scss";
 
-type ProductCardProps = {
-  product: Product;
-};
+const ProductCard = ({ product, onAddToCart }: ProductCardProps) => {
+  const handleAddToCart = () => {
+    onAddToCart(product);
+  };
 
-const ProductCard = ({ product }: ProductCardProps) => {
   return (
-    <AppLink key={product.id} to="/">
-      <AppBox className="spa-product-card">
-        <AppBox className="spa-product-card__img">
-          <AppBox
-            alt={product.name}
-            className="spa-product-card__img-name"
-            component="img"
-            src={product.image}
-          />
-          <AppBox className="spa-product-card__description">
-            <AppTypography>{product.description}</AppTypography>
-          </AppBox>
-        </AppBox>
-        <AppBox>
-          <AppTypography variant="caption">{product.name}</AppTypography>
-        </AppBox>
-        <AppBox className="spa-product-card__footer">
-          <AppTypography className="spa-product-card__footer-price">
-            {formatPrice(product.price)}
-          </AppTypography>
-          <AppButton size="small" variant="shadow">
-            <AddIcon />
-            <AppTypography translationKey="productCard.add" />
-          </AppButton>
+    <AppBox className="spa-product-card">
+      <AppBox className="spa-product-card__img">
+        <AppBox
+          alt={product.name}
+          className="spa-product-card__img-name"
+          component="img"
+          src={product.image}
+        />
+        <AppBox className="spa-product-card__description">
+          <AppTypography>{product.description}</AppTypography>
         </AppBox>
       </AppBox>
-    </AppLink>
+      <AppBox>
+        <AppTypography component={AppLink} to="/" variant="caption">
```

Comment:
Why are you decided to move link there? Maybe it would be better user experience to wrap into link img+name?

Reply:
We have button `Add to cart` inside of a `ProductCard` and if link wraps entire `ProductCard` - by clicking on a button we also click on a link, so user is navigated which is not desired behaviour when user clicks on a `Add to card` button. Probably it can be solved with `stopPropagation`, but that is in general a bad practice to have button inside of a link. So for now I made product title as a link.

Reply:
I mean img+title , not all card

Reply:
Sorry for misunderstanding, I added link around img and title